tests/integration/pilot/vm/vm_test.go

```diff
@@ -15,77 +15,62 @@
 package vm
 
 import (
+	"fmt"
 	"testing"
 	"time"
 
+	"istio.io/istio/pkg/test/framework/label"
+
+	"istio.io/istio/pkg/test/framework/components/namespace"
+
 	"istio.io/istio/pkg/config/protocol"
 	"istio.io/istio/pkg/test/framework"
 	"istio.io/istio/pkg/test/framework/components/echo"
 	"istio.io/istio/pkg/test/framework/components/echo/echoboot"
-	"istio.io/istio/pkg/test/framework/components/istio"
-	"istio.io/istio/pkg/test/framework/components/namespace"
-	"istio.io/istio/pkg/test/framework/components/pilot"
-	"istio.io/istio/pkg/test/framework/resource"
 	"istio.io/istio/pkg/test/util/retry"
 )
 
-var p pilot.Instance
-var ns namespace.Instance
+func TestVmOS(t *testing.T) {
```

Comment:
nit: can you add some comments about how this relates to TestVmOSPost? just about how the other one is slow so we save for postsubmit basically?

Reply:
I will! Thank you for the suggestion.